src/scripts/main.js

```diff
@@ -1,3 +1,75 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const table = document.querySelector('.field');
+const tr = document.getElementsByTagName('tr');
```

Comment:
```suggestion
const rows = document.getElementsByTagName('tr');
```
Більш зрозуміла назва